app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -293,39 +294,40 @@ class CatalogueController @Inject()(
               ServiceChartData.deploymentStability(repositoryDetails.name, maybeDataPoints.map(_.stability)),
               repositoryDetails.createdAt,
               deploymentsByEnvironmentName,
-              urlIfLeaksFound
+              urlIfLeaksFound,
```

Comment:
Maybe we could do the same to `service_info` as I mentioned in the comment few lines above for the `team_info`?